src/webpack.config.ts

```diff
@@ -143,6 +152,16 @@ function webpackConfig(args: Partial<BuildArgs>) {
 		plugins: [
 			new AutoRequireWebpackPlugin(/src\/main/),
 			new webpack.BannerPlugin(readFileSync(require.resolve(`${packagePath}/banner.md`), 'utf8')),
+			/**
+			 * We're using the banner plugin here to fix a bug with the webpack jsonp function. When the function is used
+			 * in the test bundle, the variable is not scoped at all, so it has, in our case, a function scope in node.
+			 * This little hack makes sure the function is defined by grabbing it from the window scope (jsdom).
+			 */
+			new webpack.BannerPlugin(<any> {
+				banner: `var ${jsonpFunctionName} = ${jsonpFunctionName} || window["${jsonpFunctionName}"];`,
+				raw: true,
+				test: /tests\/unit\/all\.*/
```

Comment:
what are we doing about functionals?

Reply:
Welp. The functional tests shouldn't be loading the source module, so they shouldn't be an issue. If we want to support that, we can load that `src/src.js` module for functional tests too,as well as apply this little hack. I don't know that there is any specific down side to supporting it but I left it out since its a sort of an anti-pattern I think?

Reply:
yeah that seems reasonable to me 👍